android/build.gradle

```diff
@@ -38,6 +39,12 @@ android {
 		sourceCompatibility JavaVersion.VERSION_17
 		targetCompatibility JavaVersion.VERSION_17
 	}
+	buildFeatures {
+		compose true
+	}
+	composeOptions {
+        kotlinCompilerExtensionVersion = "$kotlin_compose_version"
```

Comment:
Indentation on this line and the one below it. Tabs vs spaces or something?

Reply:
Android studio to the rescue. "Reformat Code"... VSCode FTL.